Author: Sean Finney <seanius@debian.org>
Description: Fix two failing tests
 * ext/posix/tests/posix_uname.phpt: removed
 * ext/posix/tests/posix_uname_basic.phpt: backported fix from upstream.
Bug: http://bugs.php.net/bug.php?id=50982
Bug-Debian: http://bugs.debian.org/570286
--- a/ext/posix/tests/posix_uname.phpt
+++ /dev/null
@@ -1,35 +0,0 @@
---TEST--
-Test posix_uname()
---DESCRIPTION--
-Gets information about the system.
-Source code: ext/posix/posix.c
---CREDITS--
-Falko Menge, mail at falko-menge dot de
-PHP Testfest Berlin 2009-05-10
---SKIPIF--
-<?php 
-	if (!extension_loaded('posix')) { 
-        die('SKIP - POSIX extension not available');
-    }
-?>
---FILE--
-<?php
-    $uname = posix_uname();
-    unset($uname['domainname']);
-    var_dump($uname);
-?>
-===DONE===
---EXPECTF--
-array(5) {
-  ["sysname"]=>
-  string(%d) "%s"
-  ["nodename"]=>
-  string(%d) "%s"
-  ["release"]=>
-  string(%d) "%s"
-  ["version"]=>
-  string(%d) "%s"
-  ["machine"]=>
-  string(%d) "%s"
-}
-===DONE===
--- a/ext/posix/tests/posix_uname_basic.phpt
+++ b/ext/posix/tests/posix_uname_basic.phpt
@@ -25,4 +25,4 @@ Array
     [machine] => %s
 )
 ===DONE====
-  
\ No newline at end of file
+  
